File: python/TEST/task_one.py
def check_number(array, target):
    result = []
    for index in range(0, len(array)):
        for count in range(0, len(array)):
            if array[index] + array[count] == target:
                result.append(array[index])
                result[0] = array[index]
                result[1] = array[count]
    return result                    

File: java/TEST/TaskOne.java
import java.util.Arrays;
public class TaskOne{
    public static int [] numbertodivide(int[] array, int target){
        int [] result = new int [2];
        for(int count = 0; count < array.length; count++){
            for(int index = count + 1; index < array.length; index++){
                if(array[count] + array[index] == target){
                    result[0] = array[count];
                    result[2] = array[index];
                }
                
            }
        }
        return result;
    }
}




